docs/frontend/readme.md

```diff
@@ -5,6 +5,7 @@ items:
   - webpacker.md
   - preact.md
   - liquid-tags.md
+  - debugging.md
```

Comment:
After I added this, I checked the Netlify preview for the documentation and it all looks good.

![image](https://user-images.githubusercontent.com/833231/66538042-bf0f1c80-eaf0-11e9-9f8e-84f54f41f2bd.png)